kratos/utilities/mortar_utilities.h

```diff
@@ -45,6 +44,16 @@ namespace Kratos
 ///@name Kratos Classes
 ///@{
 
+/**
+ * @brief This struct is used in order to identify when using the hitorical and non historical variables
+ */
+struct MortarUtilitiesSettings
+{
+    // Defining clearer options
+    constexpr static bool SaveAsHistoricalVariable = true;
+    constexpr static bool SaveAsNonHistoricalVariable = false;
```

Comment:
why do you need both?
One is the opposite of the other

Reply:
Well, I think is more self explanatory SaveAsNonHistoricalVariable instead of !SaveAsHistoricalVariable. The idea of the struct is being self explanatory